Assets/Vimeo/Scripts/Player/VimeoPlayer.cs

```diff
@@ -386,6 +389,22 @@ private void VideoMetadataLoad(string response)
             }
         }
 
+        private void PlayLoggingComplete(string responseCode)
+        {
+            if (OnPlayLoggingComplete != null) {
+                OnPlayLoggingComplete();
+            } else {
+                Debug.Log("Play logging complete and returned with response code " + responseCode);
+            }
+        }
+
+        public void LogPlay(float watchLength, bool isEditor)
+        {
+            if (!isEditor) {
```

Comment:
Why pass this variable in? Why not just do `Application.isEditor`? Oh, is it not available in this class?

Reply:
I exposed it so we can [test what happens when `isEditor` is false](https://github.com/vimeo/vimeo-unity-sdk/blob/play-logging-feature/Assets/Tests/Play/VimeoPlayerPlayTest.cs#L170). Should we avoid this?